third-party-crawler/third-party-crawler.py

```diff
@@ -0,0 +1,77 @@
+import csv
+import requests
+import re
+
+commonPhrases = ['mild', 'severe', 'mild symptoms']
+
+def fetch_data():
+    response = requests.get('https://raw.githubusercontent.com/beoutbreakprepared/nCoV2019/master/latest_data/latestdata.csv')
+    if response.status_code == 200:
+        return response.text
```

Comment:
can add a try and except block. Also, for now its fine, but at some point should move the url to a config file or may be declare outside of the method so that we don't have to change code if the url changes.

Reply:
done here:
https://github.com/findthecluster/api-app/pull/1/commits/906ae201e013e5a237b2ae0c8185b9ddecc6eccd